### DISCUSSION TITLE
How can I move the start icon for Ppink in the middle of my screen ?

### DISCUSSION THREAD

**Discussion by northtoprak** [pubpub-zz/ppInk#181] [General]
It is the red pen with a drawing line icon.. I can't drag it to anywhere.. Thanks

**Comment by pubpub-zz** [comment]
moving this icon can be done with right click hold and drag( see https://github.com/pubpub-zz/ppInk#how-to-use)
the new position is not automatically saved : you have to confirm saving its positions in the options.
If you want only to use only keyboard or the icon in the notification, you can use disable it in the options

**Reply by pubpub-zz** [reply]
Hi @northtoprak,
is it ok ?
